src/main/java/MaxSumMain.java

```diff
@@ -1,2 +1,44 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
 public class MaxSumMain {
+
+    public static void main(String[] args) {
+        try {
+            //Read file
+            String filePath = args[0];
+            File myObj = new File(filePath);
+            Scanner myReader = new Scanner(myObj);
+            //Get dimensions
+            int size = myReader.nextInt();
```

Comment:
How does this get dimensions? Doesn't nextInt() return the next int in the string?